types/lodash/lodash-tests.ts

```diff
@@ -6892,6 +6892,8 @@ fp.now(); // $ExpectType number
     _("").template(options); // $ExpectType TemplateExecutor
     _.chain("").template(); // $ExpectType FunctionChain<TemplateExecutor>
     _.chain("").template(options); // $ExpectType FunctionChain<TemplateExecutor>
+    _.template("", {escape: null, evaluate: null, interpolate: / /}); // $ExpectType TemplateExecutor
+    _.template("", {escape: null, evaluate: / /, interpolate: null}); // $ExpectType TemplateExecutor
```

Comment:
These two tests cover all three options. I opted to not combine them into one because that's an unrealistic scenario. No one would ever disable all three at the same time. So these are two realistic scenarios:
1. just interpolating and disabling escaping and evaluating
1. just evaluating and disabling escaping and interpolating (unlikely but whatever)